crates/node_binding/binding.d.ts

```diff
@@ -371,6 +371,12 @@ export interface JsStatsWarning {
   formatted: string
 }
 
+export interface NodeFS {
+  writeFile: (...args: any[]) => any
+  mkdir: (...args: any[]) => any
```

Comment:
@h-a-n-a cc

Reply:
This is a problem with NAPI-RS, which should be fixed there